src/test/java/Leas_Liudmila/junit/actions/AdoptPageActions.java

```diff
@@ -53,4 +62,54 @@ public void duplicatedNameCheck(String duplicatedName) {
         assertThat(myPageLL.addedPets.get(1).getText(), Matchers.equalTo(duplicatedName + "\nAVAILABLE"));
     }
 
+
+    public void adoptPet(String petName) {
+        log.info(petName + " is adopted");
+        myPageLL.pets.get(0).click();
+        myPageLL.adoptSelPetsBtnActive.click();
+        Helpers.waitInSeconds(1);
+
+    }
+
+    public void checkPetStatus(int petsToAdopt, String requiredStatus) {
+        for (int i = 0; i < petsToAdopt; i++) {
+            WebElement pet = myPageLL.petsIn.pets.get(i);
+            String status = myPageLL.getStatusOfPet(pet);
+            assertThat(status, Matchers.is(requiredStatus));
+            log.info("Pet Name and status: " + pet.getText());
+        }
+    }
+
+    public void adoptAllPet() {
+        for (WebElement pet : myPageLL.pets) {
+            pet.click();
+        }
+        myPageLL.adoptSelPetsBtnActive.click();
+        Helpers.waitInSeconds(1);
+
+    }
+
+    public void checkElementsInAdoptions(ArrayList<String> petName) {
+        String title = "Adoption:";
+        String defaultStatus = "AVAILABLE";
+        String approveBtn = "APPROVE";
+        String denyBtn = "DENY";
+
+        log.info("Adoptions In section contains title: " + addQuotes(title));
```

Comment:
Just For your information 
your method will work only for the first section 
Let's think about how can we made it better.
For example you will separate this method and the main logic will be moved into method ` checkAdoption(Adoption adp)`
In this case tou will be able to check any adoption, first, second, third if you need this in future.
So it's ok right now, just for your information